evalml/tests/automl_tests/parallel_tests/test_automl_dask.py

```diff
@@ -266,3 +273,64 @@ def test_automl_closes_engines(engine_str, X_y_binary_cls):
     )
     automl.close_engine()
     assert automl._engine.is_closed
+
+
+@pytest.mark.parametrize(
+    "engine_str",
+    engine_strs + ["sequential"],
+)
+@pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
+def test_score_pipelines_passes_X_train_y_train(
+    problem_type, engine_str, X_y_binary, X_y_regression, X_y_multi, AutoMLTestEnv
+):
+    if is_binary(problem_type):
+        X, y = X_y_binary
+    elif is_multiclass(problem_type):
+        X, y = X_y_multi
+    else:
+        X, y = X_y_regression
+
+    X_train, y_train = X[:50], y[:50]
+    X_test, y_test = X[50:], y[50:]
+    X_train, y_train = pd.DataFrame(X_train), pd.Series(y_train)
+    X_test, y_test = pd.DataFrame(X_test), pd.Series(y_test)
```

Comment:
Omega nitpick: could probably combine these lines to:
```
X_train, y_train = pd.DataFrame(X[:50]), pd.Series(y[:50])
X_test, y_test = pd.DataFrame(X[50:]), pd.Series(y[50:])
```

But might just be personal preference 😅  

Side note: This is probably a task for the larger test refactoring/cleanup PR, but I wonder if it's worth making our fixtures dataframes, since we've slowly been moving away from explicitly supporting numpy arrays lol

Reply:
@angela97lin Agreed that having our most common fixtures be numpy arrays is not ideal - maybe tests our code with unrepresentative inputs compared to what a user would pass in!